aspnet/migration/aspnetcoremigration.rst

```diff
@@ -0,0 +1,190 @@
+Migrating from ASP.NET to ASP.NET Core
+======================================
+
+In this article:
+
+.. contents:: Sections
+  :local:
+  :depth: 1  
+
+Overview
+--------
+
+This migration guide will cover the essentials on how to migrate projects from ASP.NET to ASP.NET Core. 
```

Comment:
This migration guide covers migrating an ASP.NET 5 RC1 application to ASP.NET Core.